ivy/functional/backends/jax/experimental/statistical.py

```diff
@@ -162,6 +162,19 @@ def nanmean(
     return jnp.nanmean(a, axis=axis, keepdims=keepdims, dtype=dtype, out=out)
 
 
+def nanmin(
+    a: JaxArray,
+    /,
+    *,
+    axis: Optional[Union[int, Tuple[int]]] = None,
+    keepdims: Optional[bool] = False,
+    out: Optional[JaxArray] = None,
```

Comment:
I'd like `out` as the last argument just for consistency.

Reply:
this is done